pandas/core/generic.py

```diff
@@ -3030,6 +3030,7 @@ def to_csv(
         index_label: Optional[Union[bool_t, str, Sequence[Label]]] = None,
         mode: str = "w",
         encoding: Optional[str] = None,
+        errors: str = "strict",
```

Comment:
I think @TomAugspurger mentioned this before but should add this to the end of the function; while seemingly pedantic we have a lot of users doing different things, so no real upside to change the order of arguments here

If an interest point for you there are other issues we have to make larger functions keyword only via a deprecation that could use some help

Reply:
Moved to the end, however the logic behind the argument order is now lost (just as it was for the other method/function

If you can point me to the other issues, I can see if I can help

Reply:
#27573